libr/bin/format/mz/mz.c

```diff
@@ -3,27 +3,37 @@
 #include "mz.h"
 #include <btree.h>
 
-static ut64 r_bin_mz_seg_to_paddr (const struct r_bin_mz_obj_t *bin, const ut16 segment) {
-	return (bin->dos_header->header_paragraphs + segment) << 4;
+static ut32 r_bin_mz_va_to_la (const ut16 segment, const ut16 offset) {
+	return (segment << 4) + offset;
 }
 
-int r_bin_mz_get_entrypoint (const struct r_bin_mz_obj_t *bin) {
-#if 0
-	ut16 cs = r_read_ble16 (buf + 0x16, false);
-	ut16 ip = r_read_ble16 (buf + 0x14, false);
-	ut16 pa = ((r_read_ble16 (buf + 8 , false) + cs) << 4) + ip;
-#endif
+static ut32 r_bin_mz_la_to_pa (const struct r_bin_mz_obj_t *bin, ut32 la) {
+	return la + (bin->dos_header->header_paragraphs << 4);
+}
+
+RBinAddr *r_bin_mz_get_entrypoint (const struct r_bin_mz_obj_t *bin) {
+	const MZ_image_dos_header *mz;
+	ut32 la;
+	RBinAddr *entrypoint;
+
 	if (!bin || !bin->dos_header) {
-		return -1;
+		return NULL;
+	}
+
+	mz = bin->dos_header;
+	la = r_bin_mz_va_to_la(mz->cs, mz->ip);
```

Comment:
missing space before (, use `r2i -i` to indent the file so you dont have to fix that by hand